js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
Does not help. Lets finish the discussion on how to implement and come back later to the details